html/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="../css/style.css" />
+        <title>Campeonas'22_Pharmacy school</title>
+    </head>
+    <body>
+        <header>
+            <h1>School of Pharmacy</h1>
+            <nav>
+                <a href="#importance">Importance of pharmacy profession</a>
+                <br />
+                <a href="#field">Fields in pharmacy</a> <br />
+                <a href="#quality">Qualities of a good pharmacist</a>
+            </nav>
+        </header>
+        <p>
+            Pharmacy is a course of study which produces professionals in the
+            field of drug manufacturing, production, componding, distribution
+            and use. It is act, practice or profession of preparing drugs
+            according to Doctor's prescription. The pharmacy profession is
+            cordinated by the body
+            <a href="pcn.html" title="pcn">Pharmacists Council of Nigeria (PCN)</a>.
+            <br />
+            This course is offered by over 20 universities in Nigeria. My
+            school,
+            <a href="school.html" title="COOU">
+                Chukwuemeka Odumegwu Ojukwu University
+            </a>, Anambra state happens to be one of such higher institutions and
+            has its Faculty of Pharmaceutical Sciences situated at the Igbariam
+            campus.
+            <br />
+            We have six
+            <a
+                href="department.html"
```

Comment:
Use the 'a' tag and the "href" attribute to create the anchor link, and the <a> tag with the "name" attribute to create the anchor target.
This wasn't implemented

Reply:
Thanks for the review. Do you mean something like this? name="Departments in my school of pharmacy"